tests/interfaces/test_autograd.py

```diff
@@ -35,6 +35,9 @@
 sqz_vals = np.linspace(0., 1., 5)
 
 
+pytestmark = pytest.mark.usefixtures("non_tape_mode_only")
```

Comment:
Unit test for the old core. Can be deleted soon.

Reply:
Oh that answers my question from above. We keep this for now just to be safe?